tests/testapp/tests.py

```diff
@@ -81,16 +85,44 @@ def test_admin_changelist_search(self):
                     html=False, msg_prefix=str(url)
                 )
 
-    def test_admin_autocomplete_load(self):
+    @unittest.skipIf(django.VERSION >= (3, 2), reason='Autocomplete URL for Django < 3.2')
+    def test_admin_autocomplete_load_pre_32(self):
         """
-        Test that the admin autocomplete endpoint loads.
+        Test that the admin autocomplete endpoint loads on Django 3.2 and before.
```

Comment:
these docstrings have been switched

Reply:
Have they? This test is *skipped* if Django >= 3.2, so that's for 3.1 and earlier... Let me know if I'm missing something!

This docstring is still wrong, this test doesn't actually include 3.2, it should be <3.2 not <=3.2 🤦🏻 

I'll update the reason in the `skipIf` decorator as it's a bit redundant.

Reply:
blep, it was me reading it wrong (skip if, not run if!)